examples/fields/SmartDateField.README.md

```diff
@@ -0,0 +1,68 @@
+# SmartDateField
+
+## Why
+
+The `SmartDateField` lets you worry about operations on `Date`s as objects, without worrying about parsing dates, or formatting dates.  Currently the `SmartDateField` can parse [9 date types](./DateField.spec.ts#L43-L60) without any format strings.  This lets you concentrate on business logic, without worrying about incoming date formats.  This let's you confidently perform arithmetic and comparisons in `compute`, `recordCompute`, `batchRecordsCompute`, and `validate` on `Date`s.  `SmartDateField` has an `fString` argument that controls how the date is serialized after `validate` is finished.  Flatfile users no longer have to convert in and out of dates to maintain formatting.
```

Comment:
Can we change the first "worry about" in the first sentence to "work with"?

The `SmartDateField` lets you work with operations on `Date`s as objects, without worrying about parsing dates, or formatting dates.

Reply:
Agreed with @syshinnn's comment. Also update "This let's you confidently perform arithmetic" to "This lets you confidently perform arithmetic"

Reply:
Also, looks like there are now more date types than just 9 listed here: https://github.com/FlatFilers/platform-sdk-starter/blob/feat/chrono-js-DateField/examples/fields/DateField.spec.ts#L43-L60 

Don't want to undersell ourselves 😉 we should update the number to 16!

Reply:
done